probe-rs/src/probe/cmsisdap/mod.rs

```diff
@@ -391,7 +405,7 @@ impl CmsisDap {
     fn send_jtag_sequences(
         &mut self,
         request: JtagSequenceRequest,
-    ) -> Result<Vec<u8>, CmsisDapError> {
+    ) -> Result<BitVec<u8, Lsb0>, CmsisDapError> {
```

Comment:
We could probably alias this Lsb0 bitvec for less complex types :)

Reply:
I'll do this separately, we could do better in the whole codebase